helpers/voting/polymarket.ts

```diff
@@ -14,6 +15,46 @@ export function checkIfIsPolymarket(
 
   return isPolymarket;
 }
+
+// this will only work when there are exactly 3 or more options, which should match most polymarket requests
+// it will only parse 3 options, omitting p4, which is assumed to be "too early".
+function dynamicPolymarketOptions(
+  decodedAncillaryData: string
+): DropdownItemT[] {
+  const resData = decodedAncillaryData.match(
```

Comment:
maybe we should handle the situation where they forget the spaces between the p things

Reply:
I'm sure you could do it with the regex but I'd rather jump off a cliff than figure that one out so yeah

Reply:
i actually was using replace instaed of replaceAll. i can get regex to work with no spaces, but i think this will cause other complications, it will break the formatting of the labels if they are mutliple words, so ill leave it for now